package.json

```diff
@@ -17,9 +16,10 @@
     "typescript": "^4.9.5"
   },
   "devDependencies": {
+    "@babel/core": "^7.0.0-0",
     "@babel/plugin-syntax-flow": "^7.14.5",
     "@babel/plugin-transform-react-jsx": "^7.14.9",
-    "@babel/core": "^7.0.0-0",
+    "@privacybydesign/yivi-frontend": "^0.1.0",
```

Comment:
Yivi frontend is not only needed for local development right? So in my view it was correctly mentioned as dependency.